### PR TITLE
Improvements to documentation

### PR DESCRIPTION
**Pull request by alberand**
Hey @KarlK90, thanks for open-sourcing your project! Now I'm also happy owner of YAEMK keyboard :) 

Over my build journey I found a few places where docs could be improved. Most of them are just clarifications or additional information. Commits have more info and argumentation behind.

Regarding Aliexpress links, I saw that you removed them as they were dead. I find it still quite useful as a starting point. Also, 2U keycaps you referenced are now back in stock, so they can come back. It's nice to have at least some reference to start looking. Anyway, I can drop that commit if you don't want them there.

Also, I have a few other questions which could be added but don't have answers for them yet:
- Is OLED display suppose to be soldered directly or through socket? It's not clear from the photos if it's there but your  OLED seems to be much closer to PCB than mine. Even though I used the socket as suggested in the build guide.
- Is it possible to install 2U stabilizers? I suppose no, but haven't check if it's simple modification to PCB.
- Any cables recomendation stores?

Thanks for the project!

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -273,7 +280,11 @@ as the YAEMK already supplies the display with stabilized 3.3V VDD.
 8. Solder RGB LEDs on the side facing away from you.
     Make sure to [correctly orient](#correct-orientation-of-rgb-leds) them,
     as they are not the same orientation all the time.
-    Use the interactive BOM when in doubt. `D35 to D76`
+    Use the interactive BOM when in doubt. `D35 to D76`. Check that all LEDs
+    are working by connecting the board to USB (the firmware from step 7 turns
+    on all the LEDs). Note that as LEDs are [connected in
+    chain](#correct-orientation-of-rgb-leds)  - one dead LED leads
+    to many of them being off.
```
Comment (KarlK90) [inline docs/index.md:287]:
```suggestion
    chain](#correct-orientation-of-rgb-leds)  - one dead LED leads
    to all subsequent leds being off.
```

File: docs/index.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ As soon as the MCU is in stock again this will be addressed.**
 
 After placing your order you can proceed ordering the missing parts from [LCSC](https://lcsc.com/),
 as they are both part of the same company,
-you [can save shipping costs](https://easyeda.com/forum/topic/How-to-combine-boards-JLCPCB-and-components-LCSC-into-a-common-order-cc0f3ff5fc0845839af9f37910bca742)
+you [can save shipping costs](https://www.lcsc.com/faqs?id=140)
```
Comment (KarlK90) [inline docs/index.md:165]: Great, thanks for the update!

File: docs/index.md [hunk 3]
```diff
@@ -219,6 +224,8 @@ it is the lower led in the picture.
 They shine up through the PCB,
 away from the surface the keyboard rests on.
 
+![LED chain orientation](images/yaemk_led_chain.webp)
```
Comment (KarlK90) [inline docs/index.md:227]: Thanks.

**Review by KarlK90 — CHANGES_REQUESTED** [review]
Hi @alberand, thank you for improving the docs and your interest in my YAEMK keyboard. I hope you enjoy using it! There are is one minor suggestion for your changes, everything else LGTM.

**Comment by KarlK90** [comment]
> Hey @KarlK90, thanks for open-sourcing your project! Now I'm also happy owner of YAEMK keyboard :)

Very nice! If you like you can share share some pictures of your build on reddit or the new [emk home](https://lemmy.world/c/ergomechkeyboards@lemmy.world). To your questions:

Reintroducing the aliexpress links is fine though I expect them to bit rot in the future again.

> * Is OLED display suppose to be soldered directly or through socket? 

The OLED is just pushed into the socket, not soldered. To minimize the height you can remove the black plastic spacer of the display pins and trim the pins so the display sits flush on the socket.

> * Is it possible to install 2U stabilizers? I suppose no, but haven't check if it's simple modification to PCB.

No this is not possible and there is hardly any space left on the pcb due to the customizability of the thumb-cluster.

> * Any cables recomendation stores?

Not really no, I just use plain USB-C PD cables, nothing fancy handcrafted.

> Thanks for the project!

You're welcome :-)

**Comment by alberand** [comment]
> > Hey @KarlK90, thanks for open-sourcing your project! Now I'm also happy owner of YAEMK keyboard :)
> 
> Very nice! If you like you can share share some pictures of your build on reddit or the new [emk home](https://lemmy.world/c/ergomechkeyboards@lemmy.world). To your questions:
> 

For sure, I will, I'm just working on a model in FreeCAD for the casing. If I get it to some fancy looking form, I will create PR in case you or anyone else is interested. For now, it's just bare PCB without any case or rubber feets (oh those are needed so much :D).

> Reintroducing the aliexpress links is fine though I expect them to bit rot in the future again.
> 
> > * Is OLED display suppose to be soldered directly or through socket?
> 
> The OLED is just pushed into the socket, not soldered. To minimize the height you can remove the black plastic spacer of the display pins and trim the pins so the display sits flush on the socket.
> 

aha, so I wasn't wrong that the one on photos looked much closer to the PCB :D
> > * Is it possible to install 2U stabilizers? I suppose no, but haven't check if it's simple modification to PCB.
> 
> No this is not possible and there is hardly any space left on the pcb due to the customizability of the thumb-cluster.
> 

Thanks, that's clear.

> > * Any cables recomendation stores?
> 
> Not really no, I just use plain USB-C PD cables, nothing fancy handcrafted.

I see.

**Pull request merged**